docs/pages/bssw/bssw_styling_common.md

```diff
@@ -85,14 +85,14 @@ The decision to *allow* or *require* references is one that should be agreed upo
 
 ## Nonstandard handling of Markdown
 
-The [bssw.io] site uses as custom tool to translate the Markdown in each `*.md` file into HTML.
+The [bssw.io](https://bssw.io/) site uses as custom tool to translate the Markdown in each `*.md` file into HTML.
 This tool deals with Markdown differently in several ways compared to other common Markdown renderers (such as GitHub and GitHub Pages).
-It is important to know about these differences up front to avoid problems once the `*.md` files are previewed on [preview.bssw.io] and finally published to [bssw.io].
+It is important to know about these differences up front to avoid problems once the `*.md` files are previewed on preview.bssw.io and finally published to [bssw.io](https://bssw.io/).
```

Comment:
@rinkug, these already have links as you can see at the bottom of this document:

```
<!-- Common hyperlinks/>

[bssw.io]: https://bssw.io
[preview.bssw.io]: https://preview.bssw.io
```

I will revert this commit since it is not needed.